src/components/ScriptExplorer/DirectSignatureImporter.jsx

```diff
@@ -37,21 +37,34 @@ class DirectSignatureImporter extends React.Component {
   };
 
   interaction = () => {
-    const { signatureImporter, network, inputs, outputs, walletConfig } =
-      this.props;
+    const {
+      signatureImporter,
+      network,
+      inputs,
+      outputs,
+      walletConfig,
+      extendedPublicKeyImporter,
+    } = this.props;
     const keystore = signatureImporter.method;
     const bip32Paths = inputs.map((input) => {
       if (typeof input.bip32Path === "undefined")
         return signatureImporter.bip32Path; // pubkey path
       return `${signatureImporter.bip32Path}${input.bip32Path.slice(1)}`; // xpub/pubkey slice away the m, keep /
     });
+    const policyHmac =
+      // eslint-disable-next-line react/prop-types
```

Comment:
why is it necessary here?

Reply:
I think it's because I didn't bother writing out the full prop type validation for this object and then it complained that it was declared. I kind of just wish we had typescript already here I guess, but I should probably make the proptypes more explicit in the meantime.

Reply:
understood, Yeah! Is @Beginner1166 going to be starting the typescript engine on this repo?

Reply:
Addressed in [15c6c20](https://github.com/unchained-capital/caravan/pull/296/commits/15c6c2061eb8d7a44ab8375e91d82444acbd36ff). If nothing else hopefully makes typescript conversion a bit easier.